CHANGELOG.md

```diff
@@ -8,7 +8,7 @@
 ### Added
 
 ### Fixed
-
+- disable adding address contacts to groups #5336
```

Comment:
```suggestion
- don't show "add contact manually" (by email address) on Chatmail accounts, because it's useless most of the time #5336
```

Reply:
Adapted it, but _"removed "Add contact manually" when on chatmail account"_ is already in the changed section